src/themes/cornerstone/register_success.js

```diff
@@ -0,0 +1,37 @@
+import { shopgateHideElementByClassName } from '../../modules/ShopgateHideElementByClassName';
+import { shopgateExecuteAppRelatedCode } from '../../modules/ShopgateExecuteAppRelatedCode';
+
+/**
+ * Makes register success page escape proof
+ */
+export function shopgateRegisterSuccess() {
+  shopgateHideElementByClassName('header');
+  shopgateHideElementByClassName('footer');
+  shopgateHideElementByClassName('button');
+
+  /**
+   * Shows the login page in the app
+   */
+  function shopgateAppOpenLoginForm() {
+    const commands = [
+      {
+        c: 'popTabToRoot',
+        p: { targetTab: 'main' },
+      },
+      {
+        c: 'openPage',
+        p: {
+          src: 'sgapi:register/login', targetTab: 'main',
+        },
+      },
+      {
+        c: 'showTab',
+        p: { targetTab: 'main' },
+      },
+    ];
+
+    window.SGJavascriptBridge.dispatchCommandsStringForVersion(JSON.stringify(commands), '9.0');
```

Comment:
why do you think that? Because of the code in order-placed? I copied the code from the legacy project and I think this was a special case for back then. I already executed this code with the Android app. So I guess this is something we don't need to care about. We need to refactor the  order-placed when it comes to this.

Reply:
I checked the behavior and its not working for iOS but for Android. I also asked Daniela this morning what we expect the App to do when we registered a user and I have no answer yet.

Reply:
I have added a TODO and updated the lib version. Carsten can give maybe more insights